CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 ## Unreleased
 
 * [BUGFIX] Fix inaccuracy in `e2ealerting` caused by invalid purging condition on timestamps. #117
-
+* [ENHANCEMENT] Add `--use-legacy-routes` flag to `rules` #99
```

Comment:
**suggestion:** `CHANGE` is a better fit for this. No reason to break it up into two separate entries.

```suggestion
* [CHANGE] When using `rules` commands, cortex ruler API requests will no default to using the `/api/v1/` prefix. 
The `--use-legacy-routes` flag has been added to allow users to use the original `/api/prom/` routes. #99
```